platform/rct/hal/usb_stm32/USB-CDC_device_stm32.c

```diff
@@ -1,52 +1,250 @@
-#include "USB-CDC_device.h"
```

Comment:
Sorting includes is preferential and can, in fact, cause some (admittedly not well designed) code to work.  That needs striken out of the standard if it currently exists.  It doesn't serve a purpose.